server/src/main/java/keywhiz/service/resources/automation/v2/SecretResource.java

```diff
@@ -363,22 +363,20 @@ public Iterable<SanitizedSecret> secretListingV2(@Auth AutomationClient automati
   /**
    * Retrieve listing of secrets expiring soon
    *
-   * @param time timestamp for farthest expiry to include
+   * @param notAfterInclusiveUnixEpochSeconds timestamp for farthest expiry to include
    *
-   * responseMessage 200 List of secrets expiring soon
+   * responseMessage 200 List of secret names expiring soon
    */
   @Timed @ExceptionMetered
   @Path("expiring/{time}")
   @GET
   @Produces(APPLICATION_JSON)
   @LogArguments
-  public Iterable<String> secretListingExpiring(@Auth AutomationClient automationClient, @PathParam("time") Long time) {
+  public Iterable<String> listExpiringSecretNames(@Auth AutomationClient automationClient, @PathParam("time") Long notAfterInclusiveUnixEpochSeconds) {
```

Comment:
With anything that has units or denotes a boundary (or both, in this case) I like the name to be very clear about what it is we're dealing with.